spec/fsm_redirect_helper.rb

```diff
@@ -0,0 +1,23 @@
+# Conditionally monkey-patches ApplicationController#redirect_to and calls super
+# only in tests if environment variable PRINT_REDIRECTS is set.
+# To enable: $ export PRINT_REDIRECTS=true
+# $ bundle exec rspec spec/...
+# Output looks like:
+# Redirected FROM .../app/services/flow/flow_state_machine.rb:170:in `redirect_to_step'
+#   TO: "http://127.0.0.1:50654/verify/doc_auth/link_sent"
+# To disable: $ unset PRINT_REDIRECTS
+# Handy for debugging the Flow State Machine.
+class ApplicationController
```

Comment:
If you move this into `spec/support` it will be [automatically required](https://github.com/18F/identity-idp/blob/main/spec/rails_helper.rb#L32). I also think this more thematically matches the contents of that folder.

Reply:
Done, thanks!